### PR TITLE
DLS-9710: Improve Session/request ID Logging

### PR DESCRIPTION
**Pull request by johno1985**

### CODE REVIEW

**Comment by johno1985** [comment]
rebuild

**Comment by johno1985** [comment]
rebuild

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,11 +3,11 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.15.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.18")
-addSbtPlugin("org.scalameta"     % "sbt-scalafmt"          % "2.4.6")
```
Comment (LAJW) [inline project/plugins.sbt:9]: 🤔 This doesn't resolve any versions, it's just formatting of the plugins file.

**Review by LAJW — APPROVED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (johno1985) [reply project/plugins.sbt:9]: The addition of the second % character ensures that the scala version of the plugin is the same as that defined in build.sbt

**Comment by johno1985** [comment]
rebuild

**Pull request merged**